region/src/main/java/org/motechproject/nms/region/csv/impl/StateImportServiceImpl.java

```diff
@@ -30,6 +31,7 @@ public StateImportServiceImpl(StateDataService stateDataService) {
     }
 
     @Override
+    @CacheEvict(value = {"state-code", "state-name", "state-all" }, allEntries = true)
```

Comment:
It still feels wrong to have three caches for each object.  IT would be nice if we could build the key with a prefix so there would be only one to deal with. Not a big deal though